recipes/libsolv/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "libsolv" %}
+{% set version = "0.7.3" %}
+{% set sha256 = "1ce36cbee920b1ee0f924262eb386023ef44a007ff872b442cab5b4568362c04" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/openSUSE/libsolv/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-fix-cmake-zlib.patch
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    
+  host:
+    - zlib
+
+test:
+  commands:
```

Please add some test commands. It would be good if you test the shared library compiled

---

There are binaries in `$PREFIX/bin` that could be tested as well thought at least one of them looks like it shouldn't be included in the package (`$PREFIX/bin/installcheck`).
```suggestion
  commands:
    - test -f $PREFIX/lib64/libsolv.so.1  # [unix]
    - test -f $PREFIX/lib64/libsolvext.so.1  # [unix]
```
It also might make more sense to install these in `$PREFIX/lib` rather than `$PREFIX/lib64`.